### ISSUE TITLE
Multiple mounts are broken

### ISSUE THREAD

**Issue by mikker** [mikker/passwordless#28]
`Passwordless.mounted_as` doesn't allow for multiple mounts.

**Comment by Uysim** [comment]
@mikker  I have an idea like
```ruby
passwordless_for :user 
passwordless_for :admin
```
then we can have route method like this
```ruby
user_passwordless_token_sign_in
admin_passwordless_token_sign_in
```
This concept taking from devise. If you need my help I can create pull request

**Comment by mikker** [comment]
This would require #22 and a bit more work to generate resource based routes. PRs are welcome (but be aware that I don't think this is a small task.)

**Comment by mikker** [comment]
Fixed in upcoming 1.0
